rsconf/package_data/docker/daemon.json.jinja

```diff
@@ -19,4 +19,7 @@
     "storage-opts": [
         "overlay2.override_kernel_check=true"
     ]
+    {% for k in docker.get('daemon_aux', [])  %}
```

Comment:
Invalid file, missing common on line above. I think it would actually be ideal to generate daemon.json without jinja when I see this. Just merge PKDicts and call dump_json.

Reply:
Nice catch. Thanks.

ae5db64